components/backgroundlayers/metadata/backgroundlayers.yml

```diff
@@ -0,0 +1,28 @@
+name: Background layers
+status: Verified
+SpectrumSiteSlug: https://spectrum.adobe.com/page/background-layers/
+examples:
+  - id: backgroundlayers-browsing
+    name: Browsing Contexts
+    markup: |
+      <div class="spectrum-Examples" style="justify-content: flex-start; position: relative; height: 150px;">
+        <div class="spectrum-BackgroundLayers--elevated" style="inline-size: 100px; block-size: 100px; border-radius: 10px; position: absolute; z-index: 4;">
```

Comment:
using inline styling here to keep these out of the classes

Reply:
Yep that seems like the right move 👍